selftests/functional/test_teststmpdir.py

```diff
@@ -18,6 +18,17 @@
 basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
 basedir = os.path.abspath(basedir)
 
+INSTRUMENTED_SCRIPT = """import os
+from avocado import Test
+class MyTest(Test):
+    def test1(self):
+        file = os.path.join(self.teststmpdir, 'file1.txt')
+        open(file, "w+").close()
+        path, dirs, files = os.walk(self.teststmpdir).next()
+        if len(files) != 1:
+            self.fail()
```

Comment:
And here in order to check the tmpdir is persistent between tests I'd expect slightly different test. How about creating another tmpfile in `self.teststmpdir` and then checking for 2 files in it (via `len(os.listdir(self.teststmpdir))`) and combining the simple and instrumented tests in a way that you execute:

```
avocado run simple.sh instrumented.py
```

therefor the simple one expects 1 file and the instrumented 2 files?